include/quicr/quicr_common.h

```diff
@@ -16,7 +16,7 @@ namespace quicr {
  *    end-to-end, 1400 - 119 = 1281.  A max data size of 1280 should be good
  *    end-to-end for all paths.
  */
-constexpr uint16_t MAX_TRANSPORT_DATA_SIZE = 1200;
+constexpr uint16_t MaxTransportDataSize = 1200;
```

Comment:
I generally prefer `snake_case` for values.

Reply:
Me too. I've had push back for snake case in this repo but I'm happy to rename this one. I'd rename it all if I could